packages/playwright-ct-svelte/index.d.ts

```diff
@@ -42,22 +42,23 @@ type JsonObject = { [Key in string]?: JsonValue };
 
 type Slot = string | string[];
 
-export interface MountOptions<Component extends SvelteComponent> {
+export interface MountOptions<Component extends SvelteComponent = SvelteComponent> {
   props?: ComponentProps<Component>;
   slots?: Record<string, Slot> & { default?: Slot };
   on?: Record<string, Function>;
   hooksConfig?: JsonObject;
 }
 
-interface MountResult extends Locator {
+interface MountResult<Component extends SvelteComponent> extends Locator {
   unmount(): Promise<void>;
+  rerender(options: Omit<MountOptions<Component>, 'hooksConfig'|'slots'>): Promise<void>
```

Comment:
The name of the method associated with the update has already been discussed?

I say this because the name `rerender` doesn't convince me. Perhaps something more expressive like `update` would be a better option and improve the intuitive understanding of the API.

```ts
let component = mount(<App></App>, {
   props: {
      count: 0 
   }
});

component.update({
   props: {
      count: 1,
   }
});
```

Reply:
Hi, thanks for the review! Why doesn't `rerender` convince you?

Reply:
I'm fine with the `update`, it sounds simpler to me as well.

Reply:
I created a draft PR for it: https://github.com/microsoft/playwright/pull/17972. Shall we discuss it further there?